app/notifications/validators.py

```diff
@@ -48,9 +48,9 @@ def check_service_over_daily_message_limit(key_type, service):
             service_stats = services_dao.fetch_todays_total_message_count(service.id)
             redis_store.set(cache_key, service_stats, ex=3600)
 
-        if service.message_limit - int(service_stats) <= 100:
-            current_app.logger.info(f'service {service.id} nearing daily limit'
-                                    f'{service.message_limit} - {service_stats}')
+        if (int(service_stats) / service.message_limit) * 100 > 75:
+            current_app.logger.info(f'service {service.id} daily limit reached '
+                                    f'{int(service_stats) / service.message_limit * 100}%')
```

Comment:
The logging statement should have additional formatting to limit the number of decimal places.  In Python3, (int / int) is a float.
```
>>> 75 / 350 * 100
21.428571428571427
```

Reply:
I agree. It should probably be limited to 1 or 2 decimal places in this case.

Reply:
Formatting [fstrings for percent](https://docs.python.org/3/library/string.html?highlight=expressing%20percentage) is super easy btw, just `:.2f%` 

No need to multiply by 100.

Reply:
thank you guys for the feedback.